src/Distribution/Server/Pages/Recent.hs

```diff
@@ -120,32 +196,30 @@ recentRevisionsURL :: URL
 recentRevisionsURL = "/packages/recent/revisions.html"
 
 
-recentFeed :: Users -> URI -> UTCTime -> [PkgInfo] -> RSS
-recentFeed users hostURI now pkgs = RSS
+recentFeed :: PaginatedConfiguration -> Users -> URI -> UTCTime -> [PkgInfo] -> RSS
```

Comment:
I don't know if this is too much work, but there is [a standard for paginated feeds](https://www.rfc-editor.org/rfc/rfc5005#section-3). Seems to me like there is no way for the feed reader to show the next page using the current iteration of this PR, is that correct?

Reply:
Yup. Text-RSS doesn't have a way to pass multiple Links at the top level or give those links a "rel" attribute. I'd be more then willing to improve Text-RSS but that won't be quick.